identity-service/src/txRelay.js

```diff
@@ -24,37 +24,32 @@ async function delay (ms) {
   return new Promise(resolve => setTimeout(resolve, ms))
 }
 
+async function getGasPrice (req, web3) {
+  let gasPrice = parseInt(await web3.eth.getGasPrice())
+  if (isNaN(gasPrice) || gasPrice > HIGH_GAS_PRICE) {
+    req.logger.info('txRelay - gas price was not defined or was greater than HIGH_GAS_PRICE', gasPrice)
```

Comment:
i think having the logger log the HIGH_GAS_PRICE would be helpful too

Reply:
@vicky-g so the reason i didn't print out HIGH_GAS_PRICE and MIN_GAS_PRICE out is because these are constants that don't change. seemed redundant to print the same thing over and over? 🤷

Reply:
@vicky-g i'm going to merge this pr but i'll open a follow up pr to change some stuff. if you feel strongly about this change i can make in the other pr